Dockerfile-ubuntu2404-dpdk2311

```diff
@@ -0,0 +1,42 @@
+FROM         ubuntu:24.04
+LABEL        org.opencontainers.image.authors="PcapPlusPlus <pcapplusplus@gmail.com>"
+CMD          bash
+ARG          DPDK_VERSION="23.11.3"
+
+# Configure timezone to avoid user input during build
+ENV TZ=US/Pacific
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Required system packages
+RUN apt-get update && apt-get -y install \
+  apt-utils \
+  build-essential \
+  ccache \
+  cmake \
+  curl \
+  git \
+  libnuma-dev \
+  libpcap-dev \
+  meson \
+  pkg-config \
+  python3-pip \
+  tar \
+  wget \
+  xz-utils \
+  net-tools \
+  tcpreplay \
+  python3-venv \
+  python3-dev \
+  kmod
+
+RUN python3 -m venv .venv \
```

Comment:
IMO either we use a venv everywhere or we don't use it but we should move to something consistent.

There is no strong need to have a .venv in a container so up to you :)

Reply:
In Ubuntu 24.04 we can't install anything without venv because we get this error: https://www.reddit.com/r/Ubuntu/comments/1eo7zyt/help_with_installing_python_packages_on_ubuntu/?rdt=33258

Otherwise I agree, there's no need for venv...

Reply:
You can just add --break-system-packages

Reply:
ah ok, let me try